fun.py

```diff
@@ -172,6 +172,25 @@ def off(self):
             pixels = [(0, 0, 0)] * numLEDs
             yield (pixels, 0.1, False)
 
+    def flower(self):
+        while True:
+            #variable for yellow  petals
+            petal_color = (255,233,0)
+            #set background to purple
+            pixels = [(149,44,201)] * numLEDs
+            for j in patt['flower']['pistil']:
+                #set pistil to brown
+                pixels[j] = (66,28,19)
+            for j in patt['flower']['left']:
+                pixels[j] = (petal_color)
```

Comment:
petal_color is already a tuple, this will make it a tuple of a tuple

Reply:
I see because it would be ((255,233,0))